frontend/src/model/DeviceInfo.js

```diff
@@ -16,4 +16,8 @@ export default class DeviceInfo {
     this.channels = channels;
     this.extras = extras;
   }
+
+  static getProperties() {
+    return ["name", "serialNumber", "status", "ip", "channels"];
```

Comment:
any reason we're only returning these particular properties? i see this gets called in `DeviceDetailsConciseTable.jsx` and but then `DeviceDetailsConciseRow.jsx` has a case for `lastCommunication` and the default for `extras`?
if it's about returning more concise info maybe a rename to `getConciseProperties` is in order?

Reply:
When I was discussing this with @felixlapierre we agreed to remove those from the options to display until we had a better idea of what's going on with the Device Details page, since so much is still undetermined. I'm hesitant to remove the options for the Row because nothing is locked in yet and I don't want to have to rewrite it later.

Personally, it doesn't make sense for last communication to be in the core details of the device. It will still need to be displayed, yes, but personally, I don't find that it should be with the name/IP/serial number. Might make more sense to slap it in with the Status, TBH 

I agree with the rename tho, change coming soon

Reply:
913cc52